### PR TITLE
Bump in edifice: ign-msgs7

### PR DESCRIPTION
**Pull request by chapulina**
Part of https://github.com/ignition-tooling/release-tools/issues/362.

The script also updated some `ign-fuel-tools5` that was left in the `main` branch.

Supersedes https://github.com/ignitionrobotics/ign-fuel-tools/pull/152/

### CODE REVIEW

File: include/ignition/fuel_tools/FuelClient.hh [hunk 1]
```diff
@@ -177,7 +177,7 @@ namespace ignition
       /// \brief Remove a model from ignition fuel
       /// \param[in] _id The model identifier.
       /// \return Result of the delete operation
-      /// Deprecate this function in ign-fuel-tools5. DeleteUrl
+      /// Deprecate this function in ign-fuel-tools6. DeleteUrl
```
Comment (nkoenig) [inline include/ignition/fuel_tools/FuelClient.hh:180]: Should this function already be deprecated?

Comment (chapulina) [reply include/ignition/fuel_tools/FuelClient.hh:180]: It sounds like we never got to it on v5. I'll open a PR targeting v6 now.

Comment (chapulina) [reply include/ignition/fuel_tools/FuelClient.hh:180]: See https://github.com/ignitionrobotics/ign-fuel-tools/pull/159

**Comment by chapulina** [comment]
All CI is green. I'll merge assuming I have @nkoenig 's approval :pray:

**Pull request merged**